src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4053,7 +4054,6 @@ def webapp_up(cmd, name=None, resource_group_name=None, plan=None, location=None
     with ConfiguredDefaultSetter(cmd.cli_ctx.config, True):
```

Comment:
Actually this whole block should be removed - since core-cli changed the logic on how to set local & global defaults.
 
& I also don't understand this fix - Plan like RG & webapp should have some defaults. However, whenever a user passes an argument for any of the parameters that should override the defaults. i.e values passed by user should override the defaults - otherwise we need a default for plan.

Reply:
Are you sure this block should be removed? Whenever I remove this block, the defaults are no longer set, so it seems like it is setting the defaults. Do you know what the new way to set local defaults is?